src/definitions/modules/dimmer.less

```diff
@@ -256,4 +256,96 @@ body.dimmable > .dimmer {
   background-color: @simpleInvertedEndBackgroundColor;
 }
 
+/*--------------
+     Captions
+---------------*/
+
+.caption.dimmer {
+  height: auto;
+}
+.ui[class*="bottom caption"].dimmer{
+  top: auto !important;
+  bottom: 0;
+}
+.ui[class*="center caption"].dimmer{
+  top:50% !important;
+  transform: translateY(-50%);
+  -webkit-transform: translateY(calc(-50% - .5px));
+}
+
+.ui[class*="center caption"].dimmer.transition[class*="fade up"].in {
+  animation-name: fadeInUpCenter;
+}
+.ui[class*="center caption"].dimmer.transition[class*="fade down"].in {
+  animation-name: fadeInDownCenter;
+}
+.ui[class*="center caption"].dimmer.transition[class*="fade up"].out {
+  animation-name: fadeOutUpCenter;
+}
+.ui[class*="center caption"].dimmer.transition[class*="fade down"].out {
+  animation-name: fadeOutDownCenter;
+}
+.ui[class*="center caption"].dimmer.bounce.transition {
+  animation-name: bounceCenter;
+}
+@keyframes fadeInUpCenter {
+  0% {
+    opacity: 0;
+    transform: translateY(-40%);
+    -webkit-transform: translateY(calc(-40% - .5px));
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(-50%);
+    -webkit-transform: translateY(calc(-50% - .5px));
+  }
+}
+@keyframes fadeInDownCenter {
+  0% {
+    opacity: 0;
+    transform: translateY(-60%);
+    -webkit-transform: translateY(calc(-60% - .5px));
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(-50%);
+    -webkit-transform: translateY(calc(-50% - .5px));
+  }
+}
+@keyframes fadeOutUpCenter {
+  0% {
+    opacity: 1;
+    transform: translateY(-50%);
+    -webkit-transform: translateY(calc(-50% - .5px));
+  }
+  100% {
+    opacity: 0;
+    transform: translateY(-45%);
+    -webkit-transform: translateY(calc(-45% - .5px));
+  }
+}
+@keyframes fadeOutDownCenter {
+  0% {
+    opacity: 1;
+    transform: translateY(-50%);
+    -webkit-transform: translateY(calc(-50% - .5px));
+  }
+  100% {
+    opacity: 0;
+    transform: translateY(-55%);
+    -webkit-transform: translateY(calc(-55% - .5px));
+  }
+}
+@keyframes bounceCenter {
+  0%, 20%, 50%, 80%, 100% {
+    transform: translateY(-50%);
+    -webkit-transform: translateY(calc(-50% - .5px));
+  }
+  40% {
+    transform: translateY(calc(-50% - 30px));
+  }
+  60% {
+    transform: translateY(calc(-50% - 15px));
+  }
+}
```

Comment:
Please add a blank line here and it's all good for me 😃

Reply:
> Please add a blank line here and it's all good for me

Where?  (and curiously: why?) (i remember somebody was telling me once not to insert too much whitespace :wink:

Reply:
Where ?
```less
@keyframes bounceCenter {
  0%, 20%, 50%, 80%, 100% {
    transform: translateY(-50%);
    -webkit-transform: translateY(calc(-50% - .5px));
  }
  40% {
    transform: translateY(calc(-50% - 30px));
  }
  60% {
    transform: translateY(calc(-50% - 15px));
  }
}
/* !!! NEW LINE HERE !!! */
.loadUIOverrides();
```

Why ? Well, just to be consistent with other less files actually.

Reply:
ok then